<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вывести данные</string>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Печать</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Категория</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="16">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Добавить данные</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QDateEdit" name="dateEdit">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Дата поставки</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Вывести таблицу</string>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Печать в PDF</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Название</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID категории</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Отчет</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Соединение с бд</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Графический отчет</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
